.github/workflows/ci.yml

```diff
@@ -99,6 +100,9 @@ jobs:
       - name: Build cedar-java-hello-world
         working-directory: ./cedar-java-hello-world
         run: bash config.sh && ./gradlew build
+      - name: Run cedar-java-hello-world
+        working-directory: ./cedar-java-hello-world
+        run: ./gradlew run
```

Comment:
```suggestion
        run: bash config.sh && ./gradlew run
```

Reply:
I think the issue is actually this line in the `build.gradle` file:
```
application {
  mainClassName = 'samplelib.SampleRunner'
}
```

`SampleRunner` isn't actually a class (`SampleJavaClass` is, but it has no main method). I think the intention was to run the example code via the tests, so I've changed this to `./gradlew test`. Possibly I will also need the config step -- will try that if the build fails.